### DISCUSSION TITLE
RC71 Build image has failed

### DISCUSSION THREAD

**Discussion by RSickenberg** [directus/directus#6023] [Archive]
Hello, 

First of all, many thanks for this project, we are currently investigating to use it on a middle office for our company.

Since I'm actively testing it for real usage, I'm aware of the changes and I saw the RC71 is not on docker hub since the GitHub action has failed (https://github.com/directus/directus/runs/2730215507?check_suite_focus=true).

I'm also experiencing this issue with MariaDB: https://github.com/directus/directus/issues/5999

Let me know if you would a PR to help you with this!

Best regards,
Romain

**Comment by rijkvanzanten** [comment]
@paescuj @t7tran thoughts? 🙂

**Reply by rijkvanzanten** [reply]
> Let me know if you would a PR to help you with this!

The answer is always yes 😄

**Comment by paescuj** [comment]
@rijkvanzanten Will take a look at this in 2 hours

**Reply by paescuj** [reply]
@rijkvanzanten Could you please manually rerun the GitHub action just for once? Wasn't able to reproduce it locally. If it still fails for a second time I need to take a closer look at this and you may want to revert the changes in the meantime.

**Reply by rijkvanzanten** [reply]
@paescuj Yes!

Thought the error doesn't sound like a one-off quirk to me 🤔 

```
npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /directus
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/directus'
npm ERR!  [Error: EACCES: permission denied, access '/directus'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/directus'
npm ERR! }
```

https://github.com/directus/directus/runs/2740605884?check_suite_focus=true

**Reply by t7tran** [reply]
@rijkvanzanten checking right now. I noticed the rc.71 wasn't pullable yesterday but didn't think of this.

**Reply by rijkvanzanten** [reply]
Thanks @t7tran! Seems like a permissions issue in the docker copying

**Reply by t7tran** [reply]
@rijkvanzanten please see PR #6044

**Reply by rijkvanzanten** [reply]
Saw, approved, and merged it 🙂
